src/netbase.cpp

```diff
@@ -68,6 +68,16 @@ std::string GetNetworkName(enum Network net)
     assert(false);
 }
 
+std::vector<std::string> GetNetworkNames()
+{
+    std::vector<std::string> names;
+    for (int n = Network::NET_IPV4; n != Network::NET_INTERNAL; ++n) {
+        names.emplace_back(GetNetworkName(static_cast<Network>(n)));
+    }
```

Comment:
nit: I'd slightly prefer a more generic approach here for the loop that doesn't make any assumptions about the position of concrete enums:

```suggestion
    for (int n = 0; n < Network::NET_MAX; ++n) {
        enum Network network = static_cast<enum Network>(n);
        if (network == Network::NET_UNROUTABLE || network == Network::NET_INTERNAL) continue;
        names.emplace_back(GetNetworkName(network));
    }
```
(untested)

Reply:
Yes, that's better. Thanks!

Reply:
Done, and moved it next to GetNetworksInfo() as the 2 functions are similar.